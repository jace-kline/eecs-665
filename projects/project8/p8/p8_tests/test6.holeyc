int a;

int main() {
    a = 6;
    a = a + a;
    a = a + a;
    return a;
}