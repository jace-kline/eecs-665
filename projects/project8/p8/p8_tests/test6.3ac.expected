[BEGIN GLOBALS]
a
[END GLOBALS]
[BEGIN main LOCALS]
tmp0 (tmp)
tmp1 (tmp)
[END main LOCALS]
main:       enter main
            [a] := 6
            [tmp0] := 12
            [a] := 12
            [tmp1] := 24
            [a] := 24
            setret 24
lbl_0:      leave main
